subsys/bluetooth/services/hrs.c

```diff
@@ -51,14 +52,23 @@ LOG_MODULE_REGISTER(hrs);
 	(BT_GATT_PERM_READ | BT_GATT_PERM_WRITE))			\
 
 static uint8_t hrs_blsc;
+static sys_slist_t hrs_cbs = SYS_SLIST_STATIC_INIT(&hrs_cbs);
 
 static void hrmc_ccc_cfg_changed(const struct bt_gatt_attr *attr, uint16_t value)
 {
 	ARG_UNUSED(attr);
 
+	struct bt_hrs_cb *listener;
+
 	bool notif_enabled = (value == BT_GATT_CCC_NOTIFY);
 
 	LOG_INF("HRS notifications %s", notif_enabled ? "enabled" : "disabled");
+
+	SYS_SLIST_FOR_EACH_CONTAINER(&hrs_cbs, listener, node) {
```

Comment:
The sys_slist API is not thread safe according to its documentation. Do we need to worry about it?

Reply:
If this is an issue here, then its also an issue there: https://github.com/zephyrproject-rtos/zephyr/blob/f027b55d026e0626905c7d4def36468f67dd923b/subsys/bluetooth/host/scan.c#L1564-L1572 and possibly in other places where same approach is used.

Reply:
> The sys_slist API is not thread safe according to its documentation.

The Bluetooth API is also not (completely) thread safe either ;)

Reply:
ok :ok_hand:
I'll resolve this thread then.